script/config-center/config.txt

```diff
@@ -12,8 +13,9 @@ transport.thread-factory.boss-thread-size=1
 transport.thread-factory.worker-thread-size=8
 transport.shutdown.wait=3
 service.vgroup_mapping.my_test_tx_group=default
+service.default.grouplist=127.0.0.1:8091
 service.enableDegrade=false
-service.disable=false
+service.disableGlobalTransaction=false
 client.rm.async.commit.buffer.limit=10000
```

Comment:
please add client.log.exceptionRate=100
and also script->client->spring->application.properties, application.yml